packages/schemas/src/schemaComponents.ts

```diff
@@ -108,6 +119,19 @@ export const richTextInputConfigSchema = Type.Object({
 	label: Type.Optional(Type.String()),
 	help: Type.Optional(Type.String()),
 });
+export const relationBlockConfigSchema = Type.Object(
+	{
+		// "relationshipConfig" is the config 'around' the related values
+		relationshipConfig: Type.Object({
+			label: Type.Optional(Type.String()),
+			help: Type.Optional(Type.String()),
+			pubType: Type.Optional(Type.String()),
+			component: Type.Enum(InputComponent),
```

Comment:
hmm, the more i think about it I'm not sure if having the `pubType` on the `config` column is very nice. I will be very annoying to update users if they either archive, change, or delete a `PubType` that they still have a Form element that only accepts pubs of that type. 
I think for now it could be fine, but ideally i think it being (yet another) column would be better, but i don't really want to make that call on my own! (@3mcd would appreciate your input)

If we do go with this config setup, could we maybe make the `pubType` -> `pubTypes`, ie an array instead? i could foresee us in the future allow multiple types of pubtypes for the same field (maybe both a figure and a plain image for instance), and it might save us a migration.